### PR TITLE
Fix #2080-App crash for compress action fixed.

### LINKED ISSUE #2080: App crashing on performing compress action.
**Actual Behaviour**

Once the compress action has been performed on an image, the compressed image is saved in the "Phimpme-Compress" folder. Now on navigating to that folder and trying to compress the image again, the app crashes.

**Expected Behaviour**

The app should not crash.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sauravvishal8797**
Fixed #2080
Changes: Fixed app crash for compressing images from compress folder.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -2636,6 +2636,7 @@ public void folderSelected(String path) {
                 bottomSheetDialogFragment.setSelectAlbumInterface(new SelectAlbumBottomSheet.SelectAlbumInterface() {
                     @Override
                     public void folderSelected(String path) {
+                        Log.i("lklklklkl", path);
```
Comment (codedsun) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: @sauravvishal8797 Please chck this!

File: app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java [hunk 2]
```diff
@@ -123,13 +134,109 @@ public void onClick(DialogInterface dialog, int which) {
...
+                bitmap.recycle();
+                bitmap = scaledBitmap;
+
+                System.gc();
+            } else {
+                bitmap = BitmapFactory.decodeStream(in);
+            }
+            in.close();
+
+            //Log.d(TAG, "bitmap size - width: " + bitmap.getWidth() + ", height: " +
```
Comment (codedsun) [inline app/src/main/java/org/fossasia/phimpme/editor/CompressImageActivity.java]: @sauravvishal8797 Remove this, I think!

**Comment by sauravvishal8797** [comment]
@codedsun Committed the changes, please review.

**Comment by sauravvishal8797** [comment]
@mariobehling Resolved the conflicts, please review and merge this.

**Pull request merged**